test/topology_custom/suite.yaml

```diff
@@ -10,12 +10,17 @@ run_first:
   - test_raft_recovery_basic
   - test_group0_schema_versioning
   - test_tablets_migration
+  - test_zero_token_nodes_topology_ops
```

Comment:
Use the `skip_mode` marker instead of this.

Reply:
Marking tests with `skip_in_debug` is redundant if they are also marked with `run_in_dev`. I'll just remove the `skip_in_debug` markers.